chrysom/store.go

```diff
@@ -22,7 +24,7 @@ type Pusher interface {
 	RemoveItem(ctx context.Context, id, owner string) (model.Item, error)
 }
 
-type Listener interface {
```

Comment:
renamed to `ListenerInterface` because it collided with the listener option `Listener`